Delta/Pause Menu/GridMenuViewController.swift

```diff
@@ -66,7 +66,7 @@ extension GridMenuViewController
         self.collectionView?.dataSource = self.dataSource
                 
         let collectionViewLayout = self.collectionViewLayout as! GridCollectionViewLayout
-        collectionViewLayout.itemWidth = 90
+        collectionViewLayout.itemWidth = 80
```

Comment:
```suggestion
        collectionViewLayout.itemWidth = (ExperimentalFeatures.shared.gameScreenshots.isEnabled || ExperimentalFeatures.shared.showStatusBar.isEnabled) ? 80 : 90
```

This change will affect people who don't turn on these experimental features, so it must be locked behind an Experimental Features check